packages/hint-manifest-icons/README.md

```diff
@@ -0,0 +1,231 @@
+# Manifest has icons (`manifest-icons`)
+
+`manifest-icons` requires that the icons specified in the Web App Manifest file
+have a `192x192` and `512x512` size icon
+with valid type either `png` or `jpg`.
+
+## Why is this important?
+
+Icons are important for the user of your web app.
+
+When a user adds your site to their home screen, you can define
+a set of icons for the browser to use. These icons are used in
+places like the home screen, app launcher, task switcher, splash screen, etc.
+
+From Google's [Web App Manifest Guide][google-guide],
```

Comment:
Probably a good idea to include and lead with the MDN section for this as it contains more information about cross-browser support: https://developer.mozilla.org/en-US/docs/Web/Manifest#icons.

Reply:
I'm not a big fan of copy pasting from other websites into the readme so we might want to rethink this section entirely before we publish it. Don't think it should be blocking. What do you think?

Reply:
Yeah, I'm okay leaving it for this PR and fine-tuning the wording, etc. in a follow-up change before publishing.

Reply:
okay then, leaving it for this PR. To be continued in the follow-up PR.